filer/static/filer/js/base.js

```diff
@@ -21,5 +20,36 @@ var Cl = window.Cl || {};
         if (Cl.Toggler) {
             new Cl.Toggler();
         }
+
+        $(document).on('click', '.dropdown-menu-checkboxes', function (clickEvent) {
+            clickEvent.stopPropagation();
+        });
+
+        // Temporary stuff here, please do not review
```

Comment:
please write appropriate comments like ``mocking the action buttons to work in frontend UI``